docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+services:
+  gltf-blender-io:
+    build:
+      context: .
+    # image: ux3d/gltf-blender-io
```

Comment:
In order to make this (https://github.com/KhronosGroup/glTF-Blender-IO/pull/269#discussion_r257638495) work, please uncomment this line.

Reply:
Not sure I follow?
The docker-compose.yml file is for building the docker image and running tests locally.
If the `ux3d/gltf-blender-io` image is autobuild on dockerhub, it may save the initial step of building locally, but that is really only the first time or if you force build: `docker-compose up --build`

Docker hub will build based on the `Dockerfile`

Reply:
Hm… I just used `docker-compose build` and then pushed the resulting image which seemed to work fine after I restored this line.

Reply:
Ahh ok, you should use automated builds on docker hub instead though, so that the image is build on every git push to master. Automated builds on docker hub will build from `Dockerfile` not from `docker-compose.yml`, so having both with same registry name will likely cause confusion.

Reply:
I see – thanks for clearing this up! I will tend to that on the morrow.

Reply:
Okay, it **would** be best if we could implement this directly on `khronosgroup/gltf-blender-io`. It's strange when it has to go through different organizations (like us) all the time. How can we organize this? @McNopper

Reply:
We will discuss this at the 3D Formats group tomorrow. Basically, everything has to be hosted at Khronos and/or people from Khronos have to have access rights.

Reply:
We didn't manage to get to it on this week's call, but maybe we can sort through things here... what is required to create the `khronosgroup/gltf-blender-io` image?  If that's purely a Docker thing, I see no problem with that and we can ask the Khronos admins to set things up.

If it's connected directly to CircleCI somehow, keep in mind I cannot log into CircleCI due to their permissions model, and it would be up to others like UX3D, Julien, and Ed to manage it. Alternatively `travis-ci.org` is fine, while `travis-ci.com` is not. I don't know what other alternatives may exist, or the pros/cons.

Reply:
I'd be happy to help setting things up on docker hub. I think the main requirement is that it needs access to the KhronosGroup github team - you could always invite me as a member if you dare :P

Feel free to contact me directly if that helps (pelle `at` brand3d `dot`com). I'm located in Denmark CET timmezone. I also normally hang around in the #gltf channel on KhronosDevs slack.

Reply:
Thanks, following up in Slack 👍